spot-ml/src/main/scala/org/apache/spot/SuspiciousConnectsScoreFunction.scala

```diff
@@ -12,13 +12,17 @@ class SuspiciousConnectsScoreFunction(topicCount: Int,
 
     val zeroProb = Array.fill(topicCount) { 0d }
 
-    // If either the ip or the word key value cannot be found it means that it was not seen in training.
-    val topicGivenDocProbs = ipToTopicMixBC.value.getOrElse(ip, zeroProb)
-    val wordGivenTopicProbs = wordToPerTopicProbBC.value.getOrElse(word, zeroProb)
-
-    topicGivenDocProbs.zip(wordGivenTopicProbs)
-      .map({ case (pWordGivenTopic, pTopicGivenDoc) => pWordGivenTopic * pTopicGivenDoc })
-      .sum
+    if(word == "word_error"){
```

Comment:
where is this value coming from?
presumably, we should have a val (string constant) somewhere, not a hardwired value in multiple places